### DISCUSSION TITLE
SMTP Injection SendEmailCommand

### DISCUSSION THREAD

**Discussion by nicosuerohynds** [aws/aws-sdk-js-v3#5802] [Q&A] [answered]
Hello. I would like to know if the input parameters in SendEmailCommand are sanitized or require us to sanitize to avoid SMTP Injection. Thanks

**Comment by iann0036** [comment, marked as answer]
SES performs server-side validation of customer input on the SendEmail API, but only for validity - i.e. the source field, in addition to being well-formatted, must also be a verified identity.

The SES team recommend customers apply best-practices and sanitise any external input themselves before passing it to any downstream system including SES.
